app/app.js

```diff
@@ -3,11 +3,7 @@ import Resolver from './resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
-let App;
-
-Ember.MODEL_FACTORY_INJECTIONS = true;
```

Comment:
Why does this go away?

Reply:
looks like it's no longer needed. https://github.com/emberjs/ember.js/pull/15204